extensions/oplogPopulator/OplogPopulatorConfigValidator.js

```diff
@@ -8,6 +8,7 @@ const joiSchema = joi.object({
     numberOfConnectors: joi.number().required().min(1),
     prefix: joi.string().optional(),
     probeServer: probeServerJoi.default(),
+    connectorsUpdateCronRule: joi.string().default('*/5 * * * * *'),
```

Comment:
I actually used the same value used for all backbeat recurring tasks 5 seconds.

The issue is not in getting the list of connectors but the list of buckets assigned to a connector, this can not be done once as the values change with each update.

So with each new bucket to add (or remove) from a connector we have to get that list of buckets and update it.

When having multiple updates at the same time, the update tasks might get the wrong list (not updated), add the tasks bucket and then configure the connector. This can lead into not taking into consideration the other updates.